inat.label.py

```
@@ -114,10 +111,10 @@ def escape_rtf(text):
         '\\"': '\\u34\'',           #  Does not work, yet - see https://www.perplexity.ai/search/If-the-RTF-gOdEwtp2TnmQZoPfQGqpsQ
         'µ': '\\u181?',
         '×': '\\u215?',
-        '“': '\\ldblquote ',
-        '”': '\\rdblquote ',
-        '‘': '\\lquote ',
-        '’': '\\rquote ',
+        '"': '\\ldblquote ',
+        '"': '\\rdblquote ',
```

Ruff (0.8.2)

Dictionary key literal `'"'` repeated

(F601)